app/views/surveys/_order_list.html.erb

```diff
@@ -0,0 +1,30 @@
+<head>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
Ich vermute, dass das Skript nicht funktioniert, da Sie hier (und an anderen Stellen) zum Einen einen zweiten <HEAD> Teil einfügen und zum Anderen JQuery (nochmals) laden.
Für Skript, das ausgeführt werden soll, nachdem die Seite und JQuery geladen ist, können Sie den Teil in einen content_for-Befehl setzen, s. Beispiel unter https://github.com/PingoUPB/PINGOWebApp/blob/master/app/views/home/index.html.erb#L39 

JQuery UI könnten Sie in einem solchen Abschnitt auch laden (es wird HTML eingefügt, Skripte kommen also nochmal in <script>, s. Beispiel); unser Vorschlag wäre aber Sie nutzen eine "einfachere" Variante des Draganddrop-Sortierens (da hier sonst die Stundenten auf ihre Handys das recht große JQuery UI Paket laden müssten), wie z. B. http://rubaxa.github.io/Sortable/

Wir würden natürlich beim Mergen auch alles nochmal gemeinsam durchgehen und Kleinigkeiten, die uns auffallen optimieren oder anpassen. 
Arbeiten Sie hieran noch weiter? Dann würden wir noch ein paar Tage abwarten - wenn Sie neue Commits in Ihr Repository schieben wird hier der Pullrequest ja automatisch aktualisiert ortable